code/config.h

```diff
@@ -393,7 +393,7 @@
 
 
 /* Pool MVT Configuration -- see <code/poolmv2.c> */
-/* FIXME: These numbers were lifted from mv2test and need thought. */
+/* TODO: These numbers were lifted from mv2test and need thought. */
```

Comment:
m. This and other constants in config.h are not justified here or in the design.  See https://github.com/Ravenbrook/mps/blob/1d940552cf33ceb46c807288d953e7e308988824/design/config.txt#to-document .  Raise an issue?

Reply:
@UNAA008 adds: TODOs especially need to be comprehensible to people who come across them without deep knowledge of the thing they annotate.  Form part of the doc describing how to use TODOs.

Reply:
1. Raise: #176 and also added references to that in 19d7fafaa4924512370bfb95e50a1570caed03b8
2. Raise: https://github.com/Ravenbrook/mps/issues/140#issuecomment-1443203684